src/main/java/fi/metatavu/metaform/server/files/FileUploadServlet.java

```diff
@@ -60,6 +91,8 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws S
       ServletOutputStream servletOutputStream = resp.getOutputStream();
       try {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Extract this nested try block into a separate method. ([squid:S1141](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1141))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.metaform-api%3Ametaform-api&pullRequest=57&issues=AWWZ3fCn7khxiKmGi_oS&open=AWWZ3fCn7khxiKmGi_oS)

[ISSUE_KEY]: AWWZ3fCn7khxiKmGi_oS